### PR TITLE
Scaling changed

### PR DESCRIPTION
**Pull request by borcuttjahns**
Everything should be working as you had described - the only weirdness is that now a couple cell types have negative values in their response curve (slightly) and that figure 6A and B look a bit worse due to some very small specificities in the beginning (experimentally)

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
Can you take a look at Fig. 1B-E? The shaded regions maybe disappeared?

**Comment by aarmey** [comment]
Besides that one point this looks pretty good! There is still some sort of baseline issue with the data in some cases. For example, in Fig. 4E the data is shifted off from 0, and it certainly would fit (and therefore should have been scaled) with an intercept closer to 0. I don't think it can be a background subtraction issue, because the intercept term should account for that, so I'm at a loss...

**Comment by borcuttjahns** [comment]
I handled the first issue mentioned but the second issue I think is a product of a phenomena shown better in figure S5. Since we're grouping T-helpers and T-helper variants together, and the model picks out big differences between memory T-helpers and others, it seems as though here the intercept plays a necessarily bigger role due to the MSE calculation that goes into regression calculations being particularly harsh against outliers (so mem T-helpers seem to dominate the weighting, and offset the smaller T-helper accuracy). If you want to scale these cell types separately and individually I think it would fix the problem, but it's your call if that's good practice or not.

**Review by aarmey — APPROVED** [review]
Excellent! This looks much better. I think we can finish up the reviewer response with this version. Let me know if you think there are other big changes we need. Otherwise, I'll start proofing both that and the manuscript.

**Comment by borcuttjahns** [comment]
I think that should do it for the code. We just need to make the appropriate edits in the text for the revised scaling, and I'm getting started on drafting up all the ODE's - after that we should be set to go!

File: ckine/figures/figureCommon.py [hunk 1]
```diff
@@ -455,9 +454,13 @@ def expScaleWT(predSTAT2, predSTAT15, expSTAT2, expSTAT15, rep2=False):
 
         subExpSTAT2 = np.reshape(subExpSTAT2, (subExpSTAT2.shape[0], subExpSTAT2.shape[1], subExpSTAT2.shape[2], 1))
         subExpSTAT2 = np.tile(subExpSTAT2, (1, 1, 1, subPredSTAT2.shape[3]))
+        subExpSTAT2 = subExpSTAT2[:, 0:2, :, :]
         subExpSTAT15 = np.reshape(subExpSTAT15, (subExpSTAT15.shape[0], subExpSTAT15.shape[1], subExpSTAT15.shape[2], 1))
         subExpSTAT15 = np.tile(subExpSTAT15, (1, 1, 1, subPredSTAT15.shape[3]))
+        subExpSTAT15 = subExpSTAT2[:, 0:2, :, :]
```
Comment (aarmey) [inline ckine/figures/figureCommon.py:460]:
```suggestion
        subExpSTAT15 = subExpSTAT15[:, 0:2, :, :]
```

Is this a typo?

Comment (borcuttjahns) [reply ckine/figures/figureCommon.py:460]: Yep - grabbing that now.

Comment (borcuttjahns) [reply ckine/figures/figureCommon.py:460]: It won't change anything as it's just a redundant line

Comment (borcuttjahns) [reply ckine/figures/figureCommon.py]: Yep - just fixed.

**Pull request merged**